go-controller/pkg/ovn/controller/unidling/unidled_annoation_controller.go

```
@@ -33,10 +33,13 @@ func NewUnidledAtController(k kube.Interface, serviceInformer cache.SharedIndexI
 	}
 
 	klog.Info("Setting up event handlers for services")
-	serviceInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
+	_, err := serviceInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
 		UpdateFunc: uac.onServiceUpdate,
 	})
-
+	if err != nil {
+		klog.Fatalf("Could not add Event Handler, %v", err)
```

This should return an error because it is called by ```func (oc *DefaultNetworkController) Run(ctx context.Context) error {``` which returns an error